enterprise-suite/gotests/util/lbc/lbc.go

```diff
@@ -12,13 +14,44 @@ import (
 const localChartPath = "../../../."
 const lbcPath = "../../../scripts/lbc.py"
 
-func Install(lbcArgs, helmArgs []string) error {
+type Installer struct {
+	UsePersistentVolumes string
```

Comment:
It would be more straightforward if this was a `bool`

Reply:
Not quite so simple - I need the "not present" value here, so we don't set it at all (because a `--set` overrides a `-f`). I would have to use a pointer to bool, and set it to nil to not set. I thought a string was simpler.

Reply:
Note the tests where this is used are testing out the values file (PVC detection logic).